src/events/AddReactionEventHandler.ts

```diff
@@ -11,9 +12,11 @@ export default class AddReactionEventHandler implements EventHandler {
 
 	private readonly selectRoleHandler = new SelectRoleEventHandler();
 	private readonly resolveRequestHandler = new ResolveRequstEventHandler();
+	private readonly newRequestEventHandler: NewRequestEventHandler;
 
-	constructor( botUserId: string ) {
+	constructor( botUserId: string, internalChannels: Map<string, TextChannel> ) {
```

Comment:
This shouldn't be needed as it's statically available in `BotConfig.request.internal_channels`

Reply:
Tried replacing it, apparently not the same types. This is also how all the otehr listeners are doing it

Reply:
Ah actually nevermind, the map is not stored and only created in `MojiraBot` as it lokks loke